docs-site/docs/protocol/data-availability.md

```diff
@@ -1,3 +1,47 @@
 ---
+sidebar_position: 1
 title: Data Availability
 ---
+
+Blobs provides reliable and high-throughput Data Availability services. Before diving into exactly what this is, we'll take a quick detour through Rollup scaling solutions.
+
+## Rollup Architecture
+
+The family of Rollup scaling approaches is very broad, and includes systems such as Optimistic Rollups, ZK Rollups, Parachains, and more.
+
+The core idea behind rollups is that new, Layer 2 blockchains can be bootstrapped from mature and secure Layer 1 blockchains simply by posting the transactions of the Layer 2 blockchains onto a Layer 1 chain as raw data. Any node scanning the Layer 1 blocks for Layer 2 transactions, and executing those Layer 2 transactions in order, will achieve the same outcome as any other node which does so. Therefore, the Layer 2 blockchain, or Rollup, is as secure as the Layer 1's transaction ordering guarantees. The terms Layer 2 and Rollup may be used interchangeably.
```

Comment:
Not sure about the last sentence. L2 is a broader term and used when the scaling or extending of functionality of a L1 happens without changing that L1 but building on top of the functionality that it provides a L2. An example that is L2 but not rollup is a state or payment channel.